lib/Parser/ParseFlags.h

```diff
@@ -51,6 +51,9 @@ enum
                                      //  let/const in global scope instead of eval scope so that they can be preserved across console inputs
     fscrNoAsmJs = 1 << 25, // Disable generation of asm.js code
     fscrIsModuleCode = 1 << 26, // Current code should be parsed as a module body
-    fscrAll = (1 << 27) - 1
+
+    fscrIndirect = 1 << 27,
```

Comment:
Removed 'fscrIndirect' but left 'fscrEval' alone. Was able to use 'fscrEval' to signify direct eval call in parser while leaving the 'fscrEval' logic for bytecodegen intact (therefore no renaming). Reverted changes to EvalHelper API.


---
In reply to: [73579364](https://github.com/Microsoft/ChakraCore/pull/1377#discussion_r73579364) [](ancestors = 73579364,73578663)